src/queuetweeter.py

```diff
@@ -29,10 +29,13 @@ def send_next_message(
         # https://docs.tweepy.org/en/stable/client.html#tweepy.Client.create_tweet
         # Or "file" (byte array) which will be uploaded w/Twitter API v1
         tweet_args = message_transformer(next_message["content"])
-        if tweet_args.get("file"):
-            media = self.twitterv1.media_upload(file=tweet_args["file"])
+        if not preview_mode and (tweet_args.get("file") or tweet_args.get("filename")):
```

Comment:
I love the addition of a preview mode. Is there a test-case written for this logic?

Reply:
There was, but it wasn't great. I just improved it.